src/style.css

```diff
@@ -1,3 +1,94 @@
 body {
   margin: 0;
+  display: block;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
 }
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+ }
+
+ .logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+ }
+
+ img {
+  width: 100%;
+  height: 100%;
+  display: block;
+ }
+
+
+ ul {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  list-style: none;
+  padding-left: 0;
+ }
+
+ li {
```

Comment:
<img width="1079" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/6c14013c-6139-4ad4-9715-7c30d1832272">
follow the code style rules